### ISSUE TITLE
Write up JSON file with features metadata

### ISSUE THREAD

**Issue by priesgo** [TRON-Bioinformatics/neofox#8]

**Comment by dlaehnemann** [comment]
Yes, such a programmatic meta-annotation file would actually be great to have. Especially, if it contains some more detail than is currently provided in the [docs about Output Data](https://neofox.readthedocs.io/en/latest/03_02_output_data.html#general-information). Some of the entries there are not really understandable without reading up on the tools or manuscripts behind them (that are nicely documented on the [main overview of the docs](https://neofox.readthedocs.io/en/latest/01_overview.html)), while it would be great to get an immediate idea of how a score / rank was generated and what its interpretation is.

In any case, many thanks for your efforts!
